src/module.cpp

```diff
@@ -1,9 +1,11 @@
 #include "vtcomposite.hpp"
-#include <nan.h>
+#include <napi.h>
+#include <uv.h>
```

Comment:
Reading a little about guaranteed ABI stability across major node versions and it includes `<uv.h>` as one of the ports that does not guarantee stability. And then continues to say, if you want a stable napi include, to use `#include <node_api.h>` - Will this be an issue?

Reply:
`#include <napi.h>` is a correct header from c++ wrapper : https://github.com/nodejs/node-addon-api/blob/master/napi.h
But, I removed `#include <uv.h>` as we shouldn't use it, thanks.